docs/site/Extending-OpenAPI-specification.md

```diff
@@ -0,0 +1,154 @@
+---
+lang: en
+title: 'Extending OpenAPI Specification'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Extending-OpenAPI-specification.html
+---
+
+## OpenAPI Specification Enhancer
+
+The APIs in a LoopBack `RestApplication` are described by the
+[OpenAPI Specification(short for OAS)](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md).
+An application's OAS are mainly generated from
+[controllers](https://loopback.io/doc/en/lb4/Controllers.html) and their
+members' metadata, but we would also like to contribute specifications from
+other places. Therefore an extension point `OASEnhancerService` is created to
```

Comment:
I think  `Therefore` can be removed or `Therefore, an extension..`

Reply:
changing it to be `Therefore, `